h2o-hadoop/scripts/Jenkins/Jenkinsfile-trigger

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/groovy
+
+def TARGET_JOB = 'michalr-hadoop-docker-test'
+def DISTRIBUTION_CDH = 'cdh'
+def DISTRIBUTION_HDP = 'hdp'
+
+parallel "CDH 5.4": {
+    stage ('CDH 5.4')
```

Comment:
this can be written in nicer way. You can have just description in a map and generate parallel for example:
```
parallel (buildMatrix.findAll{it['enabled']}.collectEntries{ c ->
                        def buildName = getBuildName(c)
                        buildFactory(buildName, c)
                    })```

Look for example here: https://github.com/dmlc/xgboost/blob/master/Jenkinsfile#L9-L12